zkevm-circuits/src/evm_circuit/execution/dummy.rs

```diff
@@ -54,6 +54,14 @@ impl<F: Field, const N_POP: usize, const N_PUSH: usize, const S: ExecutionState>
         _: &Call,
         step: &ExecStep,
     ) -> Result<(), Error> {
+        // this happens if some opcodes are in the
+        // process of being implemented but are still
+        // using DummyGadget.
+        // See `bus-mapping/src/evm/opcodes.rs`
+        if step.rw_indices.len() != N_POP + N_PUSH {
+            return Ok(());
+        }
```

Comment:
Perhaps we could just panic here instead of silent?

Reply:
this one sidesteps the dummy implementations for *CALL and other ones. Otherwise we reach `unreachable` code. We can add a log warning if that happens

Reply:
I see, sounds good to me.